nuxeo-ai-core/src/test/java/org/nuxeo/ai/bulk/TensorTest.java

```diff
@@ -233,21 +231,20 @@ public void testBadDescriptor() {
 
     @Test
     public void shouldWriteHTMLAsSingleValueTensor() throws IOException {
-        String html = "<p>Step to reproduce:</p>\n" +
-                "<ul>\n\t<li>select a File document in a content view and push the Copy button</li>\n" +
-                "\t<li>push the Paste button</li>\n" +
-                "\t<li>a new document is created in the same folder:" +
-                " get its id (<tt>cd551cce-5322-4368-b045-e13086f250bb</tt> in my example)</li>\n" +
-                "\t<li>edit the new document and increment the minor version</li>\n" +
-                "\t<li>run the following SQL query against your database\n" +
-                "<div style=\"\\\"border-width:\" class=\"\\\"preformatted\"><div class=\"\\\"preformattedContent\">\n" +
-                "<pre>select * from versions where versionableid='cd551cce-5322-4368-b045-e13086f250bb';\n" +
-                " id | versionableid | created | label | description | islatest | islatestmajor \n" +
-                "--------------------------------------------------------------------------------------" +
-                "------------------------------------------------\\n" +
-                " ba10d4af-9045-48bc-be46-0c293b84b408 | cd551cce-5322-4368-b045-e13086f250bb | 2016-05-26 14:38:00.815 | | | t | f\n" +
-                "</pre>\n</div></div>\n<p>=> the label value is empty</p></li>\n</ul>\n" +
-                "\n\n<p>This value is returned when calling <tt>session.getLastVersion(docRef).getLabel()</tt>.</p>";
+        String html = "<p>Step to reproduce:</p>\n"
```

Comment:
Maybe easier to maintain if it was moved to a resource file.

Reply:
true that